src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAutoConfiguration.java

```diff
@@ -72,4 +74,15 @@ public ContainerFactory containerFactory() {
 		return new DefaultContainerFactory(deployerProperties);
 	}
 
+	@Bean
```

Comment:
Should we mark these 2 new factory methods w/ COMB to allow consumers to override if need be?